server/Dockerfile

```diff
@@ -1,25 +1,29 @@
 FROM ubuntu:22.04
 
-# Prevent interactive prompts during package installation
-ENV DEBIAN_FRONTEND=noninteractive
+ENV UV_COMPILE_BYTECODE=1
+ENV UV_LINK_MODE=copy
+ENV APP_DIR=/app/fishtest
+ENV PATH="/root/.local/bin/:$PATH"
+
+COPY entrypoint.sh /app/entrypoint.sh
+
+RUN chmod +x /app/entrypoint.sh
 
-# Install system dependencies
 RUN apt-get update && apt-get install -y \
     build-essential \
     curl \
     git \
     gnupg \
     nginx \
     pkg-config \
-    python3 \
-    python3-pip \
-    python3-venv \
     python3-magic \
     && rm -rf /var/lib/apt/lists/*
 
+RUN mkdir -p $HOME/.local/bin
+RUN curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
I'm always nervous when I see such a download of a shell script without verification of a sha or so. Arguably, one does somewhat of the same when doing an apt-get or so, but well, hopefully ubuntu updates are super safe by design.

Reply:
good point i can later add a checksum.. though at the end of the day the script downloads a binary from somewhere else too, so it's only a minor security fix. 

The binary downloaded by the script could have been modified by a third party, and while the script checks the checksum of the downloaded binary, this verification is largely pointless from a security perspective since both the checksums and binaries are hosted on GitHub releases (https://github.com/astral-sh/uv/releases) anyway, so if an attacker gets access to that they can simply replace both files

Reply:
I guess having to trust the github releases is not too bad, that's ultimately what we rely on as well. It is also probably a fairly well maintained project, so it is not bad.